icons/shredder.json

```
@@ -0,0 +1,17 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "Alirashidy",
+    "jguddas"
+  ],
+  "tags": [
+    "file",
+    "paper",
+    "tear",
+    "cut",
+    "delete"
```

Some more tags, for better search on site!
```suggestion
    "delete",
    "destroy",
    "remove",
    "erase",
    "document,",
    "destruction",
    "secure ",
    "security",
    "confidential",
    "data",
    "trash",
    "dispose",
    "disposal",
    "information",
    "waste",
    "permanent"
```